cEP-0016.md

```diff
@@ -0,0 +1,382 @@
+# Git Commit Content Inspection
+
+| Metadata |                                                  |
+| -------- | ------------------------------------------------ |
+| cEP      | 16                                               |
+| Version  | 1.0                                              |
+| Title    | Git commit content inspection                    |
+| Authors  | Kriti Rohilla <mailto:kritirohilla567@gmail.com> |
+| Status   | Implementation due                               |
+| Type     | Process                                          |
+
+## Abstract
+
+This document describes coala process rules which will be enforced by the
+`GitCommitBear`.
+
+## Introduction
+
+`GitCommitBear` at present performs a check on the content of regular
+commits made to coala that includes performing checks on the shortlog and
```

Comment:
> commits made to coala

GitCommitBear can run on any git repository ?

Reply:
It depends on what is specified in `.coafile`. Isn't it?